core/modules/system/config/system.core.json

```diff
@@ -25,7 +25,7 @@
     "field_purge_batch_size": 200,
     "file_not_normalized_schemes": [],
     "file_additional_public_schemes": [],
-    "file_default_allowed_extensions": "jpg jpeg gif png txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp mp3 mov mp4 m4a m4v mpeg avi ogg oga ogv weba webp webm",
+    "file_default_allowed_extensions": "jpg jpeg gif png txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp mp3 mov mp4 m4a m4v mpeg avi ogg oga ogv svg weba webp webm",
```

Comment:
Just a note again (its been mentioned elsewhere I'm sure) that this is not used anywhere in Backdrop); should we delete it?

Reply:
This should probably get its own issue for proper discussion. Yes, this config item caused me confusion more than once. :wink:

Reply:
Issue exists, closing.

Reply:
Why even bothering to update, if this setting's not in use, anyway?

Reply:
We will need an update hook if we're changing this setting, even if it's not in use.

Reply:
@herbdool I didn't suggest to remove it here, but to just not touch it (to not add svg, as that's pointless, anyway).